content/posts/2025/02/06/redesign/index.md

```diff
@@ -0,0 +1,76 @@
++++
+title = 'A preview of upcoming Puzzmo.com design changes'
+date = 2025-02-06T15:38:22-08:00
+authors = ["saman"]
+tags = ["design"]
+theme = "puzzmo-light"
++++
+
+Every game we launched last year came with at least one big new feature: **Bongo circles, Pile Up Poker Series, Weather Memoku's weather widget**. We launched game variants, customizable group leaderboards, time series sparklines on completion screens, game replays, and even a year-end recap. I could keep going. We've been busy since launch. Along the way, we gathered tons of valuable feedback from players about what they love and what they can't stand about Puzzmo.com.
+
+We're going to make a set of systemic changes designed to improve the core Puzzmo experience and make space for even more experimentation in the future. Let's take a look at the work we plan to roll out over the next couple of months. Note that these mock-ups may be incomplete, and everything I'm sharing here is subject to change.
+
+## The New Today Page
+
+![The new Today page](Today.png)
+
+There are loads of subtle changes here, but a few big ones worth noting:
+
+- The toolbar beneath the header that housed the calendar and your daily progress will be incorporated into the main header, giving more space for games and making it easier to find the archives.
+- Game headings have been de-emphasized to focus the page on thumbnails. There's one exception: Crossword titles are now _more_ prominent. Titles are a part of what makes each day's Crossword unique.
+- The design of variant and series headings have been brought in line with the design of standard game headings. Series will now have a progress indicator that transforms into a pager when tapped:
+  ![Series Toggle](Pile_Up_Series_toggle.png)
+
+## The New Calendar
+
+![Calendar](Calendar.png)
+
+We never planned for the calendar to play double duty as an index of games for a given day, but we found that many players used it this way. We've decided to make it serve this purpose more effectively by adding more structure and clarity to the list of games. We plan to make the index available for non-subscribers.
+
+## The New Clubs & Friends Page
+
+![Clubs and Friends](Clubs_and_Friends.png)
+
+Social presence on Puzzmo [changed a lot](https://blog.puzzmo.com/posts/2024/12/06/rm-chat/) over the past year. We removed chat, and we've been investing more in ambient and asynchronous social rather than real-time collaboration. Turns out, our players are a lot more interested in leaderboards, daily news, and profile stats than with than trying to collab on a typeshift. In the short term, as you can see fromm the mock-up above, we're making straightforward changes to simplify managing your friends and groups. Stay tuned for more experiments related to Puzzmo's social dimension later in the year.
```

Comment:
```suggestion
Social presence on Puzzmo [changed a lot](https://blog.puzzmo.com/posts/2024/12/06/rm-chat/) over the past year. We removed chat, and we've been investing more in ambient and asynchronous social rather than real-time collaboration. Turns out, our players are a lot more interested in leaderboards, daily news, and profile stats than with than trying to collab on a typeshift. In the short term, as you can see from the mock-up above, we're making straightforward changes to simplify managing your friends and groups. Stay tuned for more experiments related to Puzzmo's social dimension later in the year.
```

Typo